pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -313,7 +315,7 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 	writer := newLogWriter(stdout, stderr, opts)
 	msg := &logMessage{}
 	for {
-		if stop {
+		if stop || (limitedMode && limitedNum == 0) {
```

Comment:
I am confused why this is needed.  wont `FindTailLineStartIndex` relative to the tail option handle this?

Reply:
I read this closer, and understand if writes are still happening relative to this tail option, users may get additional log lines.